packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -49,6 +54,14 @@ export const defaultServices = {
   }): Promise<any> {
     return await Auth.confirmSignUp(username, code);
   },
+  /** @deprecated For internal use only */
+  async handleResendConfirmationCode({
+    username,
+  }: {
+    username: string;
+  }): Promise<any> {
```

Comment:
Can we remove `Promise<any>` here?

```suggestion
  }) {
```

Idea is that TS will automatically infer the type for us.